### ISSUE TITLE
The connection was terminated before a greeting could be read

### ISSUE THREAD

**Issue by Deilan** [robinrodricks/FluentFTP#628]
**FTP OS:** Unknown

**FTP Server:** Unknown

**Computer OS:** Windows 10 x64

**FluentFTP Version:** 32.4.6

After an upgrade to v. 32.4.6 I get `System.IO.IOException`: "The connection was terminated before a greeting could be read" . 
v. 32.4.5 and previous work just fine.

**Logs :**
```
# GetListingAsync("/", Auto)

# Connect()
Status:   Connecting to ***:21

# Dispose()
Status:   Disposing FtpClient object...
Command:  QUIT
Status:   Disposing FtpSocketStream...
Status:   Disposing FtpSocketStream...
```

**Comment by SpyrexDE** [comment]
Same issue :(

**Comment by jorritvenema** [comment]
Same issue.

**Comment by kreditor0815** [comment]
I do have the same issue

**Comment by MoRooz** [comment]
same issue, downgraded to 32.4.5

**Comment by robinrodricks** [comment]
Thanks for reporting this. Fixed in [32.4.7](https://www.nuget.org/packages/FluentFTP/32.4.7).
